apps/storybook/src/Slider.stories.tsx

```diff
@@ -20,19 +20,36 @@ export default {
   args: {
     thumbMode: 'inhibit-crossing',
     trackDisplayMode: 'auto',
+    orientation: 'horizontal',
   },
 } as Meta<SliderProps>;
 
+const sliderWrapper = (element: ReactNode, args: object) => {
+  return (
+    // TODO: Is there a way to avoid the hardcoded height of 400px? height of 100% does not work
+    <div
+      className='slider-wrapper'
```

Comment:
I don't believe this div really needs a class name

Reply:
**[DONE]** Thanks, removed it